src/components/ContributeAs.js

```diff
@@ -160,7 +160,7 @@ const ContributeAs = enhance(
                 </P>
                 {value.type && (
                   <P fontSize="Caption" lineHeight="Caption" color="black.500">
-                    {key === '0' ? `Personal account - ${value.email}` : capitalize(value.type)}
```

Comment:
Fixed in https://github.com/opencollective/opencollective-frontend/pull/1205/commits/e710ed6bd862a9ea84d4ec4d7d3dc9157b2443d5